README.md

```diff
@@ -174,14 +224,14 @@ const subscription = new MessageStatusSubscription({
 const response = await client.createSubscription(subscription);
 ```
 
-The response can be an `ISubscription` object when successful or an `IError` object on errors.
+The response can be an `ISubscription` object when successful or an `IError` object when an error occurs.
 
 
-### Receiving message and message status events
+### Receive message events and message status events
 
-Use the `WebhookController` class to create your webhook to receive message and message status events. The default port is `3000`.
+Use the `WebhookController` class to create your webhook so you can receive message events and message status events. The default port is `3000`.
 
-If you inform the `client`, `url`, and `channel` fields, a subscription will be created if it does not exist for these configurations.
+If you inform the `client`, `url`, and `channel` fields, a subscription will be created prefaced it does not exist for these configurations.
```

Comment:
Prefaced? Acho que rolou algum auto complete aí kkkkk

Reply:
Sim, prefaced acho que ficou ruim. Essa sua sugestão precisa mesmo quebrar em duas frases? E o "that is" é realmente necessário? E se fosse no singular, acho que "a subscription" ficaria melhor que "the subscription", mas como na verdade são duas, colocaria no plural.

"If you inform the client, url, and channel fields, a subscription will be created, unless subscriptions matching these configurations already exists."
ou se o "that is" realmente for importante, sinto que falta uma virgula.
"If you inform the client, url, and channel fields, a subscription will be created, that is, unless subscriptions matching these configurations already exists."

O que acha?

Reply:
@stefanymiguel10 é só documentação de um dos SDKs (o de NodeJS), que é totalmente opcional o uso pra uso da API, mas não deixa de ser documentação. Sua opnião/decisão? kkkkk

Reply:
Concordo.

Mudei para

`If you inform the `client`, `url`, and `channel` fields, a subscription will be created, unless a subscription matching these configuration already exists.`